rhel7/templates/csv/file_dir_permissions.csv

```diff
@@ -2,7 +2,13 @@
 /etc,gshadow,0,0,0000
 /etc,passwd,0,0,0644
 /etc,group,0,0,0644
+/etc,crontab,0,0,0600,crontab
 /etc,cron.allow,0,0,0644,cron_allow
+/etc,cron.d,0,0,0700,cron_d
+/etc,cron.daily,0,0,0700,cron_daily
```

Comment:
`/etc/cron.daily` is a directory, but this way the templating system will generate a check for a file. To generate a check for directory please put the full path to the first column and leave the second column empty, e.g.:

```
/etc/cron.daily,,0,0,0700,cron_daily
```

Other entries in this CSV file which describe directories need to be changed the similar way as well. Also change the RHEL8 CSV.